chia/_tests/cmds/test_click_types.py

```diff
@@ -37,11 +38,9 @@
 overflow_decimal = Decimal(overflow_decimal_str)
 
 
+@dataclass
 class FakeContext:
-    obj: dict[Any, Any] = {}
-
-    def __init__(self, obj: dict[Any, Any]):
-        self.obj = obj
+    obj: dict[Any, Any] = field(default_factory=dict)
```

Comment:
Ruff didn't automatically make this fix, this was my response to a warning about the mutable default